src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -930,6 +930,15 @@
   <data name="DoNotUseObsoleteKDFAlgorithmMessage" xml:space="preserve">
     <value>Call to obsolete key derivation function {0}.{1}</value>
   </data>
+  <data name="DoNotUseReferenceEqualsWithValueTypesDescription" xml:space="preserve">
+    <value>Value type typed arguments are uniquely boxed for each call to this method, therefore the result is always false.</value>
+  </data>
+  <data name="DoNotUseReferenceEqualsWithValueTypesMessage" xml:space="preserve">
+    <value>The argument for the '{0}' parameter to ReferenceEquals evaluates to the value type '{1}'. Due to value boxing, this call to ReferenceEquals will always return false.</value>
+  </data>
+  <data name="DoNotUseReferenceEqualsWithValueTypesTitle" xml:space="preserve">
+    <value>Do not use ReferenceEquals with value types</value>
```

Comment:
All the other messages end with a period.

Reply:
Yeah we are very inconsistent in whether or not periods are used in our title/message/description strings. I think we should follow @sharwell's proposal from https://github.com/dotnet/roslyn-analyzers/pull/3183#issuecomment-576724513, which is summarized by @Evangelink in https://github.com/dotnet/roslyn-analyzers/issues/1714#issuecomment-580647362

Reply:
Another place where an analyzer/fixer to enforce this would be a much better route. @Evangelink given your initial interest in tackling #1714, would you be interested in writing this analyzer/fixer? If so, we can sync up offline on the design.

Reply:
My scan showed that things seemed to be title: no period, other: yes period.

Reply:
@mavasani Yep I'd be happy to help, let's have a chat later on.